.github/install-clang-15.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -euo pipefail
+
+if [[ $CC != "clang-15" ]]; then
```

Comment:
Can we, please, have this check explicitly on github actions level? As such, I do not think this check here is not needed. I also suggest to avoid using external `CC` variable in this script and construct the required expression from the script argument I suggested above: `clang-$1`.

Reply:
Of course - your example below works perfectly.